src/app/core/components/media-table/media-table.component.html

```diff
@@ -0,0 +1,106 @@
+<div class="table" role="table" aria-label="Media table">
+  <div class="filter">
+    <input
+      class="filter-input"
+      aria-label="Filter table"
+      [formControl]="query"
+      [placeholder]="placeholder" />
+    <button
+      *ngIf="query.value"
+      class="clear-button"
+      aria-label="Clear filter input"
+      (click)="query.setValue('')">
+      <ng-icon size="20" name="ionClose" />
+    </button>
+  </div>
+  <div class="header" role="rowgroup">
```

Comment:
I recognize that you assign specific table roles but is this enough? Did you test your table implementation vs a native table? With a screen reader? That would be interesting.